src/OpenTelemetry.Exporter.Console/ConsoleExporterOptions.cs

```diff
@@ -27,5 +27,11 @@ public class ConsoleExporterOptions
         /// Gets or sets the metric export interval.
         /// </summary>
         public int MetricExportInterval { get; set; } = 1000;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to export Delta
+        /// values or not (Cumulative).
+        /// </summary>
+        public bool IsDelta { get; set; } = true;
```

Comment:
I think a boolean is fine, but just pondering another idea... what if we had an enumeration for `AggregationTemporality` and this setting was instead

```csharp
public AggregationTemporality AggregationTemporality { get; set; } = AggregationTemporality.Delta
```

Also, this brings up an interesting question of defaults. What's the sense from the spec regarding defaults for SDK provided exporters?

Reply:
+1 with `AggregationTemporality`

Reply:
I think defaults to delta makes more sense (for console, delta generates less output). A good topic for the SDK spec :)

Reply:
Agree. making it in next PR :)